config/samples/end-to-end.yaml

```diff
@@ -70,7 +70,7 @@ spec:
     spec:
       containers:
         - name: open-feature-demo
-          image: ghcr.io/open-feature/open-feature-demo:latest
+          image: ghcr.io/open-feature/playground-app:v0.1.1
```

Comment:
The old demo app is too out of date to work with the latest flagd

Reply:
Did the repo change name also?

Reply:
It looks that way, this is where I found the reference: https://github.com/open-feature/playground/blob/main/docker-compose.yaml#L4
Perhaps @beeme1mr can clarify further

Reply:
Yes, this is correct. I think `ghcr.io/open-feature/open-feature-demo:latest` was manually published and is now out of date. With the recent changes to the playground, the demo container is published as part of the CI process.